torch/linalg/__init__.py

```diff
@@ -1480,6 +1480,22 @@
   Please use :func:`torch.linalg.svdvals`, which computes only the singular values,
   instead of `compute_uv=False`.
 
+.. note:: Selection of `driver` (this keyword argument only works on CUDA inputs with cuSOLVER backend):
+
+    - When `driver` is not given or set to `None` (*default*), a heuristic of drivers will be selected based on
```

Comment:
"a heuristic of drivers will be selected matrix sizes and batch sizes of the inputs" -> " a driver will be selected automatically based on the shape of the inputs."